lms/util/authorize_lms.py

```diff
@@ -86,8 +86,11 @@ def wrapper(request, *args, user=None, **kwargs):
                 authorization_base_url
             )
 
+            state_params = dict(request.params)
+            state_params["pubid"] = request.context.h_pubid
+
             oauth_state = find_or_create_from_user(
-                request.db, state_guid, user, request.params
+                request.db, state_guid, user, state_params
```

Comment:
This is me stashing the group's `pubid` received from h's API response in the DB along with the LTI launch params, for Canvas files assignments. This is hacky and terrible but I see no other way.